Formula/sip.rb

```diff
@@ -41,13 +42,8 @@ class Sip < Formula
   end
 
   def install
-    python3 = "python3.10"
-    venv = virtualenv_create(libexec, python3)
-    resources.each do |r|
-      venv.pip_install r
-    end
-
-    system python3, *Language::Python.setup_install_args(prefix, python3)
```

Comment:
Not sure if there was a reason this was installed separate from virtualenv. The pth file should expose `sip` libraries to other formulae.